nilearn/plotting/surf_plotting.py

```diff
@@ -219,6 +219,14 @@ def plot_surf(surf_mesh, surf_map=None, bg_map=None,
         are .png, .pdf, .svg. If output_file is not None, the plot
         is saved to a file, and the display is closed.
 
+    ax: Axes instance | None
+        The axes instance to plot to. The projection must be '3d' (e.g.,
+        `plt.subplots(subplot_kw={'projection': '3d'})`).
```

Comment:
I would write: `figure, axes = plt.subplots(subplot_kw={'projection': '3d'})` where axes should be passed.